app/models/person.rb

```diff
@@ -0,0 +1,9 @@
+class Person < ActiveRecord::Base
+  has_many :locations
+  has_one :current_location, -> { order 'created_at DESC' }, class_name: "Location"
+  has_many :disricts, :through => :locations
+  has_one :current_district, :through => :current_location, :source => 'district'
```

Comment:
Line is too long. [81/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.